volatility/framework/plugins/mac/pslist_process_groups.py

```diff
@@ -0,0 +1,89 @@
+# This file is Copyright 2019 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+#
+
+import logging
+from typing import Callable, Dict, Iterable
+
+from volatility.framework import constants, interfaces, objects, contexts, exceptions
+from volatility.plugins.mac import pslist
+
+vollog = logging.getLogger(__name__)
+
+class Pslist_Process_Groups(pslist.PsList):
+    """Lists the processes present in a particular mac memory image."""
```

Comment:
This is already used in two separate existing plugins, it has to be made more accurate (across all three ideally), otherwise there's no way of knowing how this differs from plain `pslist` differs from `tasks`.

Reply:
included in the new commit.